pages/common/blahaj.md

```diff
@@ -0,0 +1,28 @@
+# BLAHAJ
+
+> A lolcat-like queer CLI tool that colorizes your input, shows flags, and prints colorful sharks.
+> More information: <https://codeberg.org/GeopJr/BLAHAJ>.
+
+- Get a list of possible flags/colors:
+
+`blahaj --flags`
+
+- Print a blahaj with lesbian colors to console:
+
+`blahaj --shark --colors lesbian`
+
+- Print a random flag with a 2x size multiplier:
+
+`blahaj --random --flag --multiplier=2`
```

Comment:
```suggestion
`blahaj --random --flag --multiplier 2`
```
Prefer no `=` if the program supports it. Documentation tells you to use `=`, but in practice most programs support space separated arguments